suite-native/transactions/src/components/TransactionsList/TransactionListItem.tsx

```diff
@@ -43,13 +53,16 @@ export const TransactionListItemValues = ({
                 <EmptyAmountText />
             ) : (
                 <Box flexDirection="row">
-                    <SignValueFormatter value={getTransactionValueSign(transaction.type)} />
+                    {!isPhishingTransaction && (
```

Comment:
It is quite strange to me that the sign is hidden even though the transaction label still says `Sent` and the icon is arrow up for send transactions. Similar for other types.